sdk/identity/identity/src/credentials/authorizationCodeCredential.browser.ts

```diff
@@ -3,7 +3,7 @@
 
 /* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
I see this in many files disabling the no-unused-vars rules. Now that you have fixed the unused locals problem, I believe we can remove this line from all the files as well. Can you give that a try?